lib/components/Card/Card.tsx

```diff
@@ -1,26 +1,20 @@
-import React, { FC } from "react";
-import { CardWrapper } from "./Card.styled";
+import styled from "styled-components";
 import { CardProps } from "./Card.types";
 
-const Card: FC<CardProps> = ({
-  bgColor = "#fff",
-  borderRadius = 8,
-  padding = 24,
-  shadow = true,
```

Comment:
By removing these props the Card appearance will be more fixed. I thought we "adopted a philosophy" of making the components more versatile (?), but in this case the Card component will become less versatile. The good thing is that we "force" the design standard (Figma) for all cards.